cli/js/fetch.ts

```diff
@@ -366,6 +488,10 @@ async function sendFetchReq(
   return (await sendAsync(dispatch.OP_FETCH, args, zeroCopy)) as FetchResponse;
 }
 
+interface NetworkError {
+  name: string;
+  message: string;
+}
```

Comment:
Not used, if you need `NetworkError` it might be good idea to address #3740 first

Reply:
I'm actually going to remove this completely, because I added it in after improperly reading the spec. Turns out that a Network Error is a type of `Response`.